admin_guide/ipsec.adoc

```diff
@@ -0,0 +1,174 @@
+[[admin-guide-ipsec]]
+= ipsec
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+IPsec is a mechanism for encrypting communication between hosts which can
+already communicate with each other via the Internet Protocol (IP). The simplest
+way of protecting traffic within an Openshift cluster is to ensure that the
+master and all nodes use IPsec to encrypt their communication.  This document
+describes the simplest way to do this.
+
+In this example, we will secure communication of an entire IP subnet from which
+the Openshift hosts receive their IP addresses. Since you are securing the
+host-to-host communication, this automatically includes all cluster management
+and pod data traffic.
```

Comment:
Management traffic is already over https, so this adds a second layer of encryption...